scripts/changelog.py

```diff
@@ -63,10 +74,14 @@ def get_changelog(commits: List[Commit]) -> str:
     help="""Get the changelog from the merge commits
 
     The script goes through the merge commits since the specified REF
-    and get the changelog entry from the commit message.
-    By now, we parse a last paragraph of the pull-request description
-    (that is contained in the commit message).
-    In the future, we will have an explicit divider.
+    and gets the changelog entry from the commit message.
+    In case no REF is specified, the latest tag is used.
+
+    There are 2 possible ways to detect a changelog entry in the message:\n
+     1) The new way, beginning and end of the entry is explicitly marked
+        using `RELEASE NOTES {BEGIN,END}`.\n
```

Comment:
I would be explicit here.
```suggestion
        using `RELEASE NOTES BEGIN` and `RELEASE NOTES END` separators.\n
```